tests/test_version.py

```diff
@@ -25,4 +25,8 @@ def test_version_format() -> None:
 
     assert (
         __version__ == installed_version
-    ), f"{__version__} <> {installed_version}, the installed version does not match with the current codebase"
+    ), (
+        f"The installed version ({installed_version}) does not match the current codebase version ({__version__})."
+        "This failure could be due to a recent version bump in the Pyiceberg library. "
+        "Please ensure you have the latest version installed by rerunning `make install` command."
+    )
```

Comment:
We can try to fix it like this:
```suggestion
    )

```